DSCResources/MSFT_xDnsServerADZone/MSFT_xDnsServerADZone.schema.mof

```diff
@@ -8,4 +8,5 @@ class MSFT_xDnsServerADZone : OMI_BaseResource
     [Write, Description("DNS Server name")] String ComputerName;
     [Write, Description("Credential used to set zone"), EmbeddedInstance("MSFT_Credential")] String Credential;
     [Write, Description("Whether the DNS zone should be available or removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Read] String CimSession;
```

Comment:
I appreciate that I'm more than a little bit late to the conversation, but why do we need to return the CimSession at all? IMHO this is an internal object/reference and should be handled internally? Just my 2p!